luigi/configuration/getter.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is there a better name than `getter.py`?

Reply:
fixed